backend/prompt_studio/prompt_studio_core/views.py

```diff
@@ -280,11 +280,12 @@ def fetch_response(self, request: HttpRequest, pk: Any = None) -> Response:
         return Response(response, status=status.HTTP_200_OK)
 
     @action(detail=True, methods=["post"])
-    def single_pass_extraction(self, request: HttpRequest) -> Response:
+    def single_pass_extraction(self, request: HttpRequest, pk: Any) -> Response:
```

Comment:
Also, the type is `uuid` in path and `Any` in the function. Can we address that as well?

Reply:
@Deepak-Kesavan  addressed